src/test/java/seedu/address/logic/commands/TaddCommandTest.java

```diff
@@ -77,7 +77,7 @@ public void equals() {
     /**
      * A default model stub that have all of the methods failing.
      */
-    private class ModelStub implements Model {
+    private static class ModelStub implements Model {
```

Comment:
hmm why does it need to be static?

Reply:
I think I changed cause intellij prompted me to do so haha. But I checked the PaddCommandTest and its one is not static so I guess should delete the static.